flood_adapt/dbs_controller.py

```diff
@@ -269,7 +269,7 @@ def get_buildings(self) -> GeoDataFrame:
         )
         fm.read()
         buildings = fm.exposure.select_objects(
-            type="ALL",
+            primary_object_type=["ALL"],
```

Comment:
while the method in hydromt-fiat says that the input should be a list, that is not the case. The input should be a string representing the primary obect type or a string "ALL" or "all" if you want to use all the object types.

Reply:
yeah, but then we should be at hydromt_fiat?